File: Languages_LinkedList/LinkedList.java
import java.util.Scanner;

public class LinkedList {
    public LinkedList next;
    public int value;
    public static LinkedList firstNode;
    public static LinkedList finalNode=null;

    public LinkedList(int data, LinkedList point) {
        value = data;
        next = point;
    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter amount of nodes: ");
        int number_of_nodes = sc.nextInt();
        int node;

        for (int i = 0; i < number_of_nodes; i++) {
            System.out.printf("Node at %d:", i);
            node = sc.nextInt();
            LinkedList n = new LinkedList(node, null);
            if (finalNode != null) {
                finalNode.next = n;
                finalNode = n;
            } else {
                firstNode = n;
                finalNode = n;
            }
        }

        System.out.print("Values: ");
        LinkedList n = firstNode;
        while (n != null) {
            System.out.print(" " + n.value);
            n = n.next;
        }
    }
}

File: Languages_LinkedList/LinkedList.mjs
import readline from 'readline/promises'
import { stdin as input, stdout as output } from 'process';

class LinkedList {
    constructor(value, next) {
        this.value = value;
        this.next = next;
    }
}

const sc = readline.createInterface({ input, output})
const amount_of_nodes = await sc.question("Enter amount of nodes: ")

let list = null

for (let i = 0; i < amount_of_nodes; i++) {
    const value = await sc.question(`Enter value at ${i}: `)
    list = new LinkedList(value, list)
}

sc.close();

for (let ptr = list; ptr != null; ptr = ptr.next) {
    console.log(` ${ptr.value}`)
}

